tests/integration/interactive_dev_test.go

```diff
@@ -142,8 +142,7 @@ var _ = Describe("odo dev interactive command tests", func() {
 			lines, err := helper.ExtractLines(output)
 			Expect(err).To(BeNil())
 			Expect(lines).To(Not(BeEmpty()))
-			Expect(lines[0]).To(Equal("The current directory already contains source code. " +
-				"odo will try to autodetect the language and project type in order to select the best suited Devfile for your project."))
+			Expect(lines[1]).To(ContainSubstring("Dev mode ran, but no Devfile was found. Initializing a Devfile in the current directory"))
```

Comment:
`odo init` says "Initializing new component" when downloading devfile, while `odo dev` and `odo deploy` say "Initializing a Devfile". Can we make these messages more consistent?